dev/io.openliberty.rest.handler.config.openapi.2.0/src/io/openliberty/rest/handler/config/openapi20/ConfigSchemaRESTHandler.java

```diff
@@ -0,0 +1,103 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.rest.handler.config.openapi20;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.eclipse.microprofile.openapi.models.OpenAPI;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.wsspi.rest.handler.RESTHandler;
+import com.ibm.wsspi.rest.handler.RESTRequest;
+import com.ibm.wsspi.rest.handler.RESTResponse;
+
+import io.smallrye.openapi.runtime.OpenApiProcessor;
+import io.smallrye.openapi.runtime.OpenApiStaticFile;
+import io.smallrye.openapi.runtime.io.Format;
+import io.smallrye.openapi.runtime.io.OpenApiSerializer;
+
+/**
+ * Displays config schema
+ */
+@Component(name = "com.ibm.ws.rest.handler.config.openapi.Config",
```

Comment:
Should this DS name be updated?

Reply:
Either updated or deleted, I'm not sure that there's a need to define a name here, I think it will default to the name of the class anyway and I don't think it appears anywhere except in the OSGi console.